wiki/en/Running-a-Server.md

```diff
@@ -19,87 +19,67 @@ permalink: "/wiki/Running-a-Server"
+Once any issues with musicians have been solved in this way, you can then investigate hosting your own Server either at home or on a cloud host such as Amazon, which may result in better latency than Servers run at home. For example, [see this guide](https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/) for using AWS Lightsail, by Jamulus user [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a) (_Facebook_)
 
 ### Bandwidth – do you have enough?
 
 Unless you plan on hosting more than about 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up), then you are unlikely to run out of bandwidth. You can read more about network requirements at different quality settings later in this document.
 
-
 ### In general
 
 - Consider using a cloud host to get better ping times if you're having problems
 
-- Any server should have at least 1.6GHz CPU frequency and 1GB RAM
-
-- Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
+- Any Server should have at least 1.6GHz CPU frequency and 1GB RAM
 
-- Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Public Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+- Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host.
 
-- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support.
+- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
+- Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and Server. There are plans to expand IPv6 support.
 
 # Server Types
 
-You can run your server in one of three "modes" (either at home or on a 3rd party host):
-
-### 1. Public
+You can run your Server in different ways (either at home or on a 3rd party host):
 
-Use this mode when you want anyone to join your server, whoever they are.   
+### 1. Registered 
 
-Your server will be listed in the Directory that clients use by default. Musicians can then discover and connect to your server.
+Your Server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. 
```

Comment:
> By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your Server. 
 
----

This needs to mention as well listing in a Custom Directory.  Something like:

> By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your Server.  You can also have your Server listed on a Custom Directory, if that better meets your needs.